mandatory/1-fix-functions.js

```diff
@@ -10,18 +10,18 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(isHappy) {
 
```

Comment:
I understand the mistake now. This function can run without an argument being added to it since there was no need for a parameter. The function "greaterThan10(num)" has a parameter as opposed to mood(), which initially didn't.